cpanfile

```diff
@@ -0,0 +1,14 @@
+requires "Class::Method::Modifiers" => "0";
```

Comment:
For the most part, you could just drop the `=> "0"` here to pick up any version of the module you require.  Explicit version requirements tend to be declared like this as well:

```perl
requires 'perl', '5.008001';    # can also declare a minimum perl version here
requires 'Class::Method::Modifiers';
test_requires 'Test::More', '0.98';   # this specific version has improved subtests support
```

See also https://metacpan.org/pod/cpanfile

Reply:
Thanks, I tidied it up